src/diffusers/models/transformers/auraflow_transformer_2d.py

```diff
@@ -405,8 +406,10 @@ def fn_recursive_attn_processor(name: str, module: torch.nn.Module, processor):
     # Copied from diffusers.models.unets.unet_2d_condition.UNet2DConditionModel.fuse_qkv_projections with FusedAttnProcessor2_0->FusedAuraFlowAttnProcessor2_0
     def fuse_qkv_projections(self):
         """
-        Enables fused QKV projections. For self-attention modules, all projection matrices (i.e., query, key, value)
-        are fused. For cross-attention modules, key and value projection matrices are fused.
+        Enables fused QKV projections. 
```

Comment:
I think this will cause test to break because it is `# Copied from` elsewhere. This would require the implementations and docs to be the same as every other occurrence